package.json

```diff
@@ -37,9 +37,6 @@
         "vsce": "^1.79.5",
         "vscode-test": "^1.4.0"
     },
-    "extensionDependencies": [
```

Comment:
If we're going to remove this, then I believe adding a debugging section to the README with some clues as to which extensions to install would be helpful?

Reply:
Good point. I'm pretty tired but I will test and document a process in the morning. It shouldn't be too difficult based on what @RSDuck said.

Reply:
Doesn't need to be particularly substantial, something like,

title: "Enabling Debugging"
description: "To make native debugging work you need an extension such as ..." (list of links to extensions)

Improvements can from there separately, I personally got it working from the [pragmagic issue](https://github.com/pragmagic/vscode-nim/issues/65) on it and it resulted in this for [nimlsp](https://github.com/saem/nimlsp/blob/master/.vscode/launch.json#L3).

Reply:
I've pushed my progress so far so that you can see what I've done. I haven't worked with VSCode extensions before. Are you suggesting we could make the `.vscode/launch.json` a part of the extension itself, just by including it in the project, rather than instructing users to create them?

Reply:
I just meant some basic instructions in the `README.md` so someone could setup their config as they needed or at least they have a starting point. What you've got already is plenty good for a starting point and should get most people on their way, nice!